extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -177,10 +179,14 @@ export class AmpConsent extends AMP.BaseElement {
    * Register a list of user action functions
    */
   enableInteractions_() {
-    this.registerAction('accept', () => this.handleAction_(ACTION_TYPE.ACCEPT));
-    this.registerAction('reject', () => this.handleAction_(ACTION_TYPE.REJECT));
+    this.registerAction('accept',
+        () => this.handleAction_(ACTION_TYPE.ACCEPT));
+    this.registerAction('reject',
+        () => this.handleAction_(ACTION_TYPE.REJECT));
     this.registerAction('dismiss',
         () => this.handleAction_(ACTION_TYPE.DISMISS));
+    this.registerAction('dismiss',
```

Comment:
'dismiss' and enter full_screen? 

As far as I tell, the API should live within the the consent-ui.js along with the `consent-ui-ready` API. Could you explain why we need this?